packages/components/src/form-token-field/suggestions-list.js

```diff
@@ -105,9 +105,32 @@ class SuggestionsList extends Component {
 					);
 
 					/* eslint-disable jsx-a11y/click-events-have-key-events */
+					
+					var label = suggestion.label;
+
+					var decodeEntities = (function() {
+						// this prevents any overhead from creating the object each time
+						var element = document.createElement('div');
+					  
+						function decodeHTMLEntities (str) {
+						  if(str && typeof str === 'string') {
+							// strip script/html tags
+							str = str.replace(/<script[^>]*>([\S\s]*?)<\/script>/gmi, '');
+							str = str.replace(/<\/?\w(?:[^"'>]|"[^"]*"|'[^']*')*>/gmi, '');
+							element.innerHTML = str;
+							str = element.textContent;
+							element.textContent = '';
+						  }
+					  
+						  return str;
+						}
+					  
+						return decodeHTMLEntities;
+					  })();
+					  suggestion.label=decodeEntities(label);
 					return (
 						<li
-							id={ `components-form-token-suggestions-${ this.props.instanceId }-${ index }` }
+							id={ `components-form-token-suggestions78-${ this.props.instanceId }-${ index }` }
```

Comment:
looks like typo `suggestions78` in id ?

Reply:
Hello ) I forgot to remove 78 from class name when I searching a function )

Reply:
I have remove 78 from class name )